testsuite/tests/properties/dynvars/test.out

```diff
@@ -1,4 +1,7 @@
 == abstract: default, concrete: default ==
+File "test.py", line 42, in AbstractNode.use_prop
+  line 42, in bind expression
+    Warning: Useless bind of dynamic var 'env'
```

Comment:
This warning is irrelevant to the goal of this testcase (which is to test that we ensure that dynamic vars stays consistenc with inheritance). I would suggest to disable this warning in `test.py` rather than letting the warning clutter the test output.